.github/workflows/build.yml

```diff
@@ -1,22 +1,18 @@
 name: Build
 on:
-  pull_request:
-    branches:
-    - master
+  pull_request: {}
   push:
     branches:
     - master
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Btw, can we just stick with latest or is there a reason to version this?

Reply:
If a newer ubuntu is in some way incompatible with what spec-prod is doing, I think we want to upgrade intentionally. This matches the example in https://w3c.github.io/spec-prod/, and if spec-prod ever changes to need a newer OS, they can update their own version number to make sure we adopt both changes at the same time.